pallets/gear/src/tests.rs

```diff
@@ -311,7 +311,7 @@ fn exited_program_zero_gas() {
 fn delayed_user_replacement() {
```

Comment:
seems that test only tests case when delayed message sent to user.
provide different test with reproducing charging when program sends a message to another program

Reply:
done. More test coverage